docs/deployment/share-s3-access-logs.md

```diff
@@ -10,11 +10,11 @@ It is possible through Cumulus to share S3 access logs across multiple S3 packag
 
 The S3 Replicator is a Node.js package that contains a simple Lambda function, associated permissions, and the Terraform instructions to replicate create-object events from one S3 bucket to another.
 
-First ensure that you have enabled [S3 Server Access Logging](../configuration/server_access_logging).
+First, ensure that you have enabled [S3 Server Access Logging](../configuration/server_access_logging).
 
-Next configure your `config.tfvars` as described in the [`s3-replicator/README.md`](https://github.com/nasa/cumulus/blob/master/tf-modules/s3-replicator/README.md) to correspond to your deployment.  The `source_bucket` and `source_prefix` are determined by how you enabled the [S3 Server Access Logging](../configuration/server_access_logging).
+Next, configure your `config.tfvars` as described in the [`s3-replicator/README.md`](https://github.com/nasa/cumulus/blob/master/tf-modules/s3-replicator/README.md) to correspond to your deployment.  The `source_bucket` and `source_prefix` are determined by how you enabled the [S3 Server Access Logging](../configuration/server_access_logging).
```

Comment:
```suggestion
Next, configure your `terraform.tfvars` as described in the [`s3-replicator/README.md`](https://github.com/nasa/cumulus/blob/master/tf-modules/s3-replicator/README.md) to correspond to your deployment.  The `source_bucket` and `source_prefix` are determined by how you enabled the [S3 Server Access Logging](../configuration/server_access_logging).
```
I updated it from `config.tfvars` to `terraform.tfvars` since the README mentions a `terraform.tfvars.sample` file